sqlalchemy-stubs/sql/elements.pyi

```diff
@@ -47,300 +58,378 @@ class ColumnElement(operators.ColumnOperators, ClauseElement):
     def base_columns(self): ...
     def proxy_set(self): ...
     def shares_lineage(self, othercolumn): ...
-    def compare(self, other, **kw): ...
+    def compare(self, other: Any, **kw: Any) -> bool: ...
     def cast(self, type_): ...
     def label(self, name): ...
     def anon_label(self): ...
 
-class BindParameter(ColumnElement):
+class BindParameter(ColumnElement[_T], Generic[_T, _V]):
     __visit_name__: str = ...
-    key: Any = ...
-    unique: Any = ...
-    value: Any = ...
+    key: str = ...
+    unique: bool = ...
+    value: Union[_V, Any] = ...
```

Comment:
Why not just `_V` here?

Reply:
This was a mistake. It should be `Optional[_V]`.